frontend/src/pages/pipelines/global/runs/ManageRunsTable.tsx

```diff
@@ -0,0 +1,173 @@
+              }
+            },
+            isDisabled: !isChecked && selectedIdsRef.current.length === 10,
+            ...(isDisabled && { tooltip: 'Can only compare up to 10 runs' }),
+          }}
+          hasRowActions={false}
+          run={run}
+        />
+      );
+    },
+    [toggleSelection],
+  );
+
+  const onSelectAll = React.useCallback(
+    (isChecked: boolean) => {
+      if (!isSelectAllChecked) {
+        selectedIdsRef.current = pageRunIds;
+      } else {
+        pageRunIds.forEach((pageRunId) => {
+          if (selectedIdsRef.current.includes(pageRunId)) {
+            selectedIdsRef.current.splice(selectedIdsRef.current.indexOf(pageRunId), 1);
+          }
+        });
+      }
+      setIsSelectAllChecked(isChecked);
+    },
+    [isSelectAllChecked, pageRunIds],
+  );
+
+  return (
+    <Flex justifyContent={{ default: 'justifyContentCenter' }} data-testid="manage-runs-table">
+      <TableBase
+        {...checkboxTableProps}
+        loading={loading}
+        page={page}
+        perPage={pageSize}
+        onSetPage={(_, newPage) => {
+          if (newPage < page || !loading) {
+            setPage(newPage);
+
+            if (isSelectAllChecked) {
+              setIsSelectAllChecked(false);
+            }
+          }
+        }}
+        selectAll={{
+          onSelect: onSelectAll,
+          selected: isSelectAllChecked,
+        }}
+        onPerPageSelect={(_, newSize) => setPageSize(newSize)}
+        itemCount={totalSize}
+        data={runs}
+        columns={pipelineRunColumns}
+        enablePagination="compact"
+        emptyTableView={<DashboardEmptyTableView onClearFilters={filterProps.onClearFilters} />}
+        toolbarContent={
+          <PipelineRunTableToolbar
+            data-testid="manage-runs-table-toolbar"
+            filterOptions={{
+              [FilterOptions.NAME]: 'Run',
+              [FilterOptions.EXPERIMENT]: 'Experiment',
+              [FilterOptions.PIPELINE_VERSION]: 'Pipeline version',
+              [FilterOptions.CREATED_AT]: 'Started',
+              [FilterOptions.STATUS]: 'Status',
+            }}
+            {...filterProps}
+          />
+        }
+        rowRenderer={rowRenderer}
+        variant={TableVariant.compact}
+        getColumnSort={getTableColumnSort({ columns: pipelineRunColumns, ...tableProps })}
+        bottomToolbarContent={
+          <ToolbarGroup>
+            <ToolbarItem>
+              <Tooltip content="Select up to 10 runs to compare">
+                <Button
+                  data-testid="manage-runs-update-button"
+                  onClick={() =>
+                    navigate(
+                      experimentsCompareRunsRoute(namespace, experimentId, selectedIdsRef.current),
+                    )
+                  }
+                  isDisabled={selectedIdsRef.current.length < 1}
+                >
+                  Update
+                </Button>
+              </Tooltip>
```

Comment:
missing the tooltip when the button is disabled. You can use isAriaDisabled so the tooltip still works in a disabled state.